### PR TITLE
docs: Generating an api key

### PR DESCRIPTION
**Pull request by arnaudgillard**
Closes #

{New pattern submission from the Hybrid Cloud team}

#### Changelog

**New**

- {New pattern on "Generating an API Key"}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/content/patterns/adding-creating-pattern/adding-creating-pattern.mdx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+<ImageComponent cols="8">
+
+![Example](images/7.jpg)
+
+</ImageComponent>
+
+
+### Button position
+
+<ImageComponent cols="8">
```
Comment (jnm2377) [inline src/content/patterns/adding-creating-pattern/adding-creating-pattern.mdx]: Can we use the do-don't components for this set of example images? You can see an example here: https://www.carbondesignsystem.com/guidelines/color/overview https://github.com/carbon-design-system/carbon-website/blob/master/src/content/guidelines/color/overview.mdx

File: src/content/patterns/deleting/deleting.mdx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+</ImageComponent>
+
+
+## Low impact
+Requiring the user to confirm deletion is generally recommeded. However in very low impact situations, such as when an ‘undo’ option is available, user confirmation may not be required.
+
+
+#### Maintainers:
+
+[Vikki Paterson](https://github.ibm.com/VICTORIA), [Arnaud Gillard](https://github.ibm.com/Arnaud-Gillard)
```
Comment (jnm2377) [inline src/content/patterns/deleting/deleting.mdx]: Can we move this section up to the top like the other ones?

File: src/data/navigation/navigation.json [hunk 3]
```diff
@@ -233,6 +233,12 @@
       },
       "overflow-content": {
         "title": "Overflow content"
+      },
+      "adding-creating-pattern": {
+        "title": "Adding or Creating"
+      },
+      "generating-an-api-key": {
+        "title": "Generating an API Key"
```
Comment (jnm2377) [inline src/data/navigation/navigation.json]: Can we add these under experimental like the other ones?

File: src/content/patterns/adding-creating-pattern/adding-creating-pattern.mdx (hunk 1 above)
Comment (arnaudgillard) [reply src/content/patterns/adding-creating-pattern/adding-creating-pattern.mdx]: Hi Josefina, that sounds like an appropriate recommendation! I'll see what I can do. I realise that two patterns were submitted on this one issue (both "generating an API key" and "adding-creating-pattern") so will add this comment to the appropriate pull request.

**Comment by arnaudgillard** [comment]
Updated page content and imagery. I have also changed the navigation so that this page lives under "experimental"

**Comment by vikkipaterson** [comment]
@jnm2377 is this waiting for action on our side?  (Arnaud is leaving today so I'm trying to gain an understanding of what actions, if any, are outstanding from our side to getting this completed :o)

**Comment by connor-leech** [comment]
Closing due to inactivity. This is blocked by a lack of dedicated visual designers to polish/approve. Will be reopened when picked up.

Ref: https://github.com/carbon-design-system/carbon-website/issues/1859

**Pull request closed without merging**